wgsl/index.bs

```diff
@@ -3739,12 +3739,22 @@ The return type of an entry point has to be of an [=Entry point IO type=], or [=
       return vec4<f32>(0.0, 0.0, 0.0, 1.0);
     }
        // OpEntryPoint Vertex %vert_main "vert_main" %(result)
```

Comment:
%(result) should be %return_value to match the example below, same with the next example.

Reply:
nicely spotted!